filters/auth/jwt_metrics.go

```diff
@@ -27,34 +28,31 @@ type (
 )
 
 func NewJwtMetrics() filters.Spec {
-	return &jwtMetricsSpec{}
+	return &jwtMetricsSpec{
+		yamlConfig: newYamlConfig[jwtMetricsFilter](64),
```

Comment:
64 here is an arbitrary cache size here. In case number of distinct yaml configurations exceeds it the caching still has value because it will hold recently parsed configurations.

Reply:
Decided currently it does not make sense to make it configurable, we can do it later if needed.